tool/tsh/tsh.go

```diff
@@ -383,6 +393,9 @@ func Run(args []string, opts ...cliOption) error {
 	ssh.Flag("cluster", clusterHelp).StringVar(&cf.SiteName)
 	ssh.Flag("option", "OpenSSH options in the format used in the configuration file").Short('o').AllowDuplicate().StringsVar(&cf.Options)
 	ssh.Flag("no-remote-exec", "Don't execute remote command, useful for port forwarding").Short('N').BoolVar(&cf.NoRemoteExec)
+	ssh.Flag("x", "Disable x11 forwarding for this request").Short('x').BoolVar(&cf.X11ForwardingDisabled)
+	ssh.Flag("X", "Enable x11 forwarding for this request").Short('X').BoolVar(&cf.X11Forwarding)
```

Comment:
So there's a flag to enable and a flag to disable? What is the behavior when neither are set?

Reply:
I'll remove `-x`, it makes sense in ssh where they have config files, but we only use flags and x11 forwarding is disabled by default.